build/release/known_translations

```diff
@@ -0,0 +1,39 @@
+# This is a mapping of names to be translated.
+# The format expected on each line should be: <GitHub ID> - <Full Name>
```

Comment:
The mapping does not cover the following ids
```
biangjuang
weixi62961
FourSpaces
mrtisttt
XorSum
fantasticKe
camper42
mans2singh
touchida
CavemanIV
QianyongY
uicosp
rhh777
huangzhir
Kyofin
huapan123456
Karsonnel
wangyepeng2
yehere
phyyou
liaoyt
runzhliu
```

Reply:
add
```
XorSum
CavemanIV
QianyongY
mattshma
```